bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -67,7 +67,7 @@ spec:
         {{- if .Values.initContainers }}
         {{- include "common.tplvalues.render" (dict "value" .Values.initContainers "context" $) | nindent 8 }}
         {{- end }}
-        {{- if .Values.auth.tls.enabled }}
+        {{- if and .Values.auth.tls.enabled (not .Values.auth.tls.sharedJKS) }}
```

Comment:
Question, don't we want to change the permissions for the shared JKS certificates as well? Wouldn't this cause issues?

Reply:
Hello @javsalgar 

The files are just read. No special permissions required for that.

Reply:
Ok